plugins/autocomplete/plugin.js

```diff
@@ -845,7 +845,7 @@
 		 */
 		createItem: function( item ) {
 			var encodedItem = encodeItem( item ),
-				itemElement = CKEDITOR.dom.element.createFromHtml( this.itemTemplate.output( encodedItem ), this.document ),
+				itemElement = CKEDITOR.dom.element.createFromHtml( CKEDITOR.tools.htmlSafeByReview(this.itemTemplate.output( encodedItem ), 'Template code using safe encoded item'), this.document ),
```

Comment:
Let's legacy convert if it's coming from outputs of templates.

Reply:
Will be legacy converted